pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go

```diff
@@ -93,20 +93,6 @@ func TestMostAllocatedScoringStrategy(t *testing.T) {
 			expectedScores: []framework.NodeScore{{Name: "node1", Score: 62}, {Name: "node2", Score: 50}},
 			resources:      defaultResources,
 		},
-		{
```

Comment:
I would keep the tests

Reply:
resources is nil is not allowed, test will fail then.

Reply:
Failing on empty/nil map isn't new and the tests already existed and were passing. Specifically, I would like to continue to have a test that ensures we don't crash if the map was nil/empty

Reply:
Make sense, add back the tests.